airbyte-cdk/python/pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
 name = "airbyte-cdk"
-version = "2.2.0"
+version = "2.2.1"
```

Comment:
you shouldn't need to bump the version. the github action wil ltake care of it

Reply:
done